src/d3a/d3a_core/sim_results/export_unmatched_loads.py

```diff
@@ -75,20 +85,28 @@ def find_unmatched_loads(self, area, indict):
             self.name_uuid_map[child.name] = child.uuid
             self.name_type_map[child.name] = child.display_type
             if child.children:
-                indict[area.name] = self.find_unmatched_loads(child, indict[area.name])
+                indict[area.name] = self.find_unmatched_loads(
+                    child, indict[area.name], all_past_markets
+                )
             else:
                 if isinstance(child.strategy, LoadHoursStrategy):
+                    curr_market = [child.parent.current_market] \
```

Comment:
Sorry for nitpicking, IMO `current_market` would be a better name.

Reply:
agreed, changing that